sources/packages/web/src/constants/message-constants.ts

```diff
@@ -9,3 +9,13 @@ export const PENDING_SIN_MESSAGE =
  */
 export const INVALID_SIN_MESSAGE =
   "StudentAid BC has found an issue with your account information. Please ensure your account information matches CRA and log back in. If you have further issues please contact us.";
+
+/**
+ * Ministry Name
+ */
+export const MINISTRY_SHORTNAME = "PSFS";
```

Comment:
There is an old cypress test (LoginMinistryObject.ts) checking that:
![image](https://user-images.githubusercontent.com/78114138/222853229-8632ca40-84d3-4da6-9c48-5dc9c8b3a216.png)
@andrewsignori-aot  @ann-aot @dheepak-aot Should we delete those tests? Maybe update them?

Reply:
Cypress tests should stay. I would not bother to do maintenance on them right now.